### PR TITLE
Adds limbflowers, and buttflower icons

### PR DESCRIPTION
**Pull request by tortellinitony**
The beautiful sprites, courtesy of moi.

### CODE REVIEW

File: code/modules/hydroponics/grown/buttflower.dm [hunk 1]
```diff
@@ -10,6 +10,7 @@
 	maturation = 8
 	production = 6
 	yield = 1
+	growing_icon = 'icons/obj/hydroponics/growing_flowers.dmi'
```
Comment (Carbonhell) [inline code/modules/hydroponics/grown/buttflower.dm:13]: isn't this already set in the parent?

File: code/modules/hydroponics/grown/limbflower.dm [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	icon_state = "seed-limb"
+	species = "limb"
+	plantname = "Replica Limb Flower"
+	product = /obj/item/weapon/reagent_containers/food/snacks/grown/limb_spawn
+	lifespan = 25
+	endurance = 10
+	maturation = 8
+	production = 6
+	yield = 2
+	growing_icon = 'icons/obj/hydroponics/growing_flowers.dmi'
```
Comment (Carbonhell) [inline code/modules/hydroponics/grown/limbflower.dm:13]: same?

**Comment by FuryMcFlurry** [comment]
would like to see images.

File: code/modules/hydroponics/grown/buttflower.dm (hunk 1 above)
Comment (tortellinitony) [reply code/modules/hydroponics/grown/buttflower.dm:13]: butt flowers were invisible before

File: code/modules/hydroponics/grown/limbflower.dm (hunk 2 above)
Comment (tortellinitony) [reply code/modules/hydroponics/grown/limbflower.dm:13]: no

**Comment by tortellinitony** [comment]
Here first stage vs. death http://puu.sh/roN17/abed9609ca.png

second stage vs. death http://puu.sh/roN4n/10031a4448.png

third stage vs. death http://puu.sh/roN6s/fdd57cd090.png

**Pull request merged**